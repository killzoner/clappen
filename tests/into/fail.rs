#[clappen::clappen(export = generate, gen_into)]
mod generate{
    struct SimpleStruct{}
}
// This macro doesn't work with no argument when gen_into is provided
fn main() {
    generate!();
}
