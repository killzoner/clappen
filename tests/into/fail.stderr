error: unexpected end of macro invocation
 --> tests/into/fail.rs:7:5
  |
1 | #[clappen::clappen(export = generate, gen_into)]
  | ------------------------------------------------ when calling this macro
...
7 |     generate!();
  |     ^^^^^^^^^^^ missing tokens in macro arguments
  |
note: while trying to match meta-variable `$prefix:literal`
 --> tests/into/fail.rs:1:1
  |
1 | #[clappen::clappen(export = generate, gen_into)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  = note: this error originates in the attribute macro `clappen::clappen` (in Nightly builds, run with -Z macro-backtrace for more info)
